backend/audit/test_excel.py

```diff
@@ -1,440 +1,508 @@
-        workbook = load_workbook(FINDINGS_TEXT_TEMPLATE, data_only=True)
-
-        for name in findings_text_field_mapping.keys():
-            self.assertIsNotNone(workbook.defined_names[name])
-
-        for name in findings_text_column_mapping:
-            self.assertIsNotNone(workbook.defined_names[name])
-
-    def test_single_findings_text_entry(self):
-        """Test that extraction and validation succeed when there is a single findings text entry"""
-        workbook = load_workbook(FINDINGS_TEXT_TEMPLATE, data_only=True)
-
-        _set_by_name(workbook, "auditee_uei", FindingsTextExcelTests.GOOD_UEI)
-        _add_entry(workbook, 0, FindingsTextExcelTests.TEST_DATA[0])
-
-        findings = extract_findings_text(workbook)
-
-        validate_findings_text_json(findings)
-
-    def test_multiple_findings_text_entries(self):
-        """Test that extraction and validation succeed when there are multiple findings text entries"""
-        workbook = load_workbook(FINDINGS_TEXT_TEMPLATE, data_only=True)
-
-        _set_by_name(workbook, "auditee_uei", FindingsTextExcelTests.GOOD_UEI)
-        for index, entry in enumerate(FindingsTextExcelTests.TEST_DATA):
-            _add_entry(workbook, index, entry)
-
-        findings = extract_findings_text(workbook)
-
-        validate_findings_text_json(findings)
-
-    def test_partial_findings_text_entry(self):
-        """Test that extraction succeeds and validation fails when there are partial findings text entries"""
-        workbook = load_workbook(FINDINGS_TEXT_TEMPLATE, data_only=True)
-
-        _set_by_name(workbook, "auditee_uei", FindingsTextExcelTests.GOOD_UEI)
-
-        entry = jsoncopy(FindingsTextExcelTests.TEST_DATA[0])
-        del entry["text_of_finding"]
-
-        _add_entry(workbook, 0, entry)
-
-        findings = extract_findings_text(workbook)
-
-        self.assertRaises(ValidationError, validate_findings_text_json, findings)
-
-    def test_findings_text_checking(self):
-        """Test that extraction succeeds and validation fails when fields are of the wrong data type"""
-        workbook = load_workbook(FINDINGS_TEXT_TEMPLATE, data_only=True)
-
-        # add valid data to the workbook
-        _set_by_name(workbook, "auditee_uei", FindingsTextExcelTests.GOOD_UEI)
-        _add_entry(workbook, 0, FindingsTextExcelTests.TEST_DATA[0])
-
-        test_cases = [
-            ("auditee_uei", 123456789123),
-            ("reference_number", 0),
-            ("contains_chart_or_table", "not a boolean"),
-            ("text_of_finding", 10.001),
-        ]
-
-        # validate that each test_case appropriately checks the data type
-        for field_name, value in test_cases:
-            with self.subTest():
-                _set_by_name(workbook, field_name, value)
-
-                findings = extract_findings_text(workbook)
-
-                self.assertRaises(
-                    ValidationError, validate_findings_text_json, findings
-                )
+import json

```

Comment:
I've noticed a few diffs look like this. Any idea why it looks like the entire file was deleted and then recreated?

Reply:
I assume it goes away if you hide the whitespace in the diff view? My guess is a line endings issue.